codetf.schema.json

```diff
@@ -87,6 +87,34 @@
           "description": "Vendor-specific properties to help storytelling",
           "additionalProperties": true
         },
+        "failureState": {
```

Comment:
Consider simply "failure" instead of "failureState". I feel like almost everything here is "state".